UPGRADE-6.3.md

```diff
@@ -0,0 +1,9 @@
+
+Administration
```

Comment:
please add this to the `UPGRADE-6.2.md` file as you are targeting the 6.2 branch :wink:

Reply:
Done :)